### PR TITLE
Reset game speed to 1 after silo death

### PR DESCRIPTION
**Pull request by gsquaredxc**
### Brief description of the changes:

Resets game speed to 1 after the game has completed. This allows the reset game timer to complete at normal speed.

### Tested Changes:
- [ ] I've tested the changes locally or with people.
- [x] I've not tested the changes.

### CODE REVIEW

**Comment by Ragnarok77-factorio** [comment]
Mh hard to label this one, but whatever.

File: maps/biter_battles_v2/game_over.lua [hunk 1]
```diff
@@ -437,6 +437,8 @@ function Public.silo_death(event)
         global.spy_fish_timeout["south"] = game.tick + 999999
         global.server_restart_timer = 150
 
+	game.speed = 1
+
```
Comment (Ragnarok77-factorio) [inline maps/biter_battles_v2/game_over.lua:441]: Why the space ? and indent is different from the rest : 
image

**Review by Ragnarok77-factorio — CHANGES_REQUESTED** [review]
Very minor thing but OK otherwise

File: maps/biter_battles_v2/game_over.lua (hunk 1 above)
Comment (Ragnarok77-factorio) [reply maps/biter_battles_v2/game_over.lua:441]: Yes, indeed, not convinced it's a good idea tbh to format the whole project, if we ever want to compare later it's gonna be horrible will all the formatting difference shown + conflicts everywhere and so on... Fine now, I can check on github too, if you over the indent, you can see if it's a tab or a space, and compare 2 existing line.

**Review by Ragnarok77-factorio — APPROVED** [review]
Was quite difficult to review this massive update, seems OK.

**Comment by clifffrey** [comment]
I worry that in some cases, perhaps some players need a lower game.speed value to not be disconnected, just when the factories are so large (i.e. not due to biter pathing, just due to the number of inserters/etc), and this might result in many players being dropped right after the game ends.

Anyhow, I'm not trying to say that this should be a blocker on this change, I just want to call out the potential risk. For related reasons, the change might be slightly safer/less-surprising if it printed something out to the console in the case where the game speed was not already 1. Relatedly, if the game speed was set to 2 or something, it is not obvious to me that it should be lowered to 1.

**Review by XVhc6A — APPROVED** [review]
I think this changes makes sense.  I've never run into this problem myself, but i have seen it complained about in discord before.

**Pull request merged**